### ISSUE TITLE
[WinterSpring] High Chief Winterspring

### ISSUE THREAD

**Issue by noxYQ** [Valhalla-Gaming/Tracker#8896]
**Describe the issue you're having**:
not drop the Questitem "Crudely-Written-Log
**Explain how you expect it work**:
Slain the Chief and loot the Item :)
**Steps to reproduce the problem**:

1. Nothing dropped :/
2. 
3. 

**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/quest=28470/high-chief-winterfall
http://www.wowhead.com/item=12842/crudely-written-log

**Comment by lavalampe77** [comment]
Yeah, 68% drop chance doesnt mean this item drops everytime. But i have killed him 10 times now and got not a single one what means the drop chance is either way too low or he doesnt drop this item what makes a very long quest chain in winterspring not obtainable.
